src/services/const/fileNames.ts

```diff
@@ -0,0 +1,5 @@
+export const fileNames = {
+    "Audi RS5": "audi_rs5",
+    "Audi RS6": "audi-rs6",
+    "Audi e-tron GT": "audi_e-tron_gt"
+}
```

Comment:
When you export a constant it needs to have a type. When you add car model type you can write something like:
```
export const fileNames: Record<CarModel, string> = {
    "Audi RS5": "audi_rs5",
    "Audi RS6": "audi-rs6",
    "Audi e-tron GT": "audi_e-tron_gt"
}
```

Reply:
Noted. Added `Record<CarModel, string>` type definition.